great_expectations/metrics/registry.py

```diff
@@ -0,0 +1,11 @@
+from typing import Final
+
+from great_expectations.metrics.domain import ColumnMap, Domain
+
+DOMAIN_NAMES: Final[dict[type[Domain], str]] = {
```

Comment:
It would be nice if we didn't have to manually edit this file every time we added a domain or metric. Instead of making these `Final` could we add an add method that we call after the domain/metric definition or automatically gets added in the metaclass by looking at a class variable?

Reply:
I removed the registry entirely. There is already a registry in the existing metrics system and the metric names are inferred entirely from the class names.